templates/persistent-volume.yaml

```diff
@@ -18,7 +18,7 @@ spec:
   hostPath:
     # Directory location on host
     path: {{ required "need to define the policy directory by using --set global.persistence.hostpath=localpath location" .Values.global.persistence.hostpath  | quote }}
-    type: Directory
+    type: DirectoryOrCreate
```

Comment:
I thought @dev-958 removed this in another PR going through review at the moment as it shouldn't be required

Reply:
I put it back in as I was premature.